test/optimus/scientific/TestOptimusScientific.wdl

```diff
@@ -1,28 +1,27 @@
 import "Optimus.wdl" as target
-import "ScientificTests.wdl" as tests
 
 workflow TestOptimusScientific {
-  Array[Array[File]] fastq_inputs
-  File whitelist
-  File tar_star_reference
-  File annotations_gtf
-  File ref_genome_fasta
-  String sample_id
-  Int expected_number_bam_records
+  # Optimus inputs
+  Array[File] r1_fastq
+  Array[File] r2_fastq
+
+  File whitelist  # 10x genomics cell barcode whitelist for 10x V2
+  File tar_star_reference  # star reference
+  File annotations_gtf  # gtf containing annotations for gene tagging
+  File ref_genome_fasta  # genome fasta file
+  String sample_id  # name of sample matching this file, inserted into read group header
 
   call target.Optimus as target {
     input:
-      fastq_inputs = fastq_inputs,
+      r1_fastq = r1_fastq,
+      r2_fastq = r2_fastq,
       whitelist = whitelist,
       tar_star_reference = tar_star_reference,
       annotations_gtf = annotations_gtf,
       ref_genome_fasta = ref_genome_fasta,
-  }
       sample_id = sample_id
-
-  call tests.TestBamRecordNumber as test {
-    input:
-      bam = target.bam,
-      expected_records = expected_number_bam_records
+  }
+  output {
+    target.*
```

Comment:
Does this work? I don't see it mentioned in the WDL spec anywhere.

Reply:
Good 'ol WDL spec. It does work: https://job-manager.mint-dev.broadinstitute.org/jobs/8b3d75b4-11c8-4322-b197-3566e8c7d45e